apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -24,7 +25,8 @@ import {
   translatedCourseOfferingDeviceCompatibilityLevels,
 } from '@cdo/apps/templates/teacherDashboard/CourseOfferingHelpers';
 import ImageInput from './ImageInput';
-
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
 const translatedNoneOption = `(${i18n.none()})`;
```

Comment:
Nit: if possible I think it might be good to maintain the space between the import statements then the constants, then to also have the `noneOption` and `renderedOptions` constants moved up here too

Reply:
For the renderedOptions, I use props in it so I dont think I can move the constant outside the component without changing to a function. Should I make this change?

Reply:
To have a record of this: we talked on Slack and decided that it would be fine either way whether the change was made or not (more of a personal styling preference than anything)